extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -538,7 +546,13 @@ export class AmpIframe extends AMP.BaseElement {
     if (box.width > 10 && box.height > 10) {
       return false;
     }
-    return true;
+    // Iframe is not tracking iframe if open with user interaction
+    if (this.isInContainer_ === undefined) {
+      this.isInContainer_ = !!closest(this.element, element => {
```

Comment:
Yeah, we need to think about it. I'd recommend "copy paste" lightbox at this point :) But it's invariant here since devs still need to remember to whitelist name here otherwise.